test/python/test_transport.py

```diff
@@ -306,190 +319,245 @@ def test_bad_geometry(self):
-        self.check_thermal_conductivity(620, 2.8e7, 0.49705, 1e-4)
-
-    def test_viscosity_supercritical(self):
-        self.check_viscosity(660, 2.2e7, 2.7129e-5, 1e-4)
-        self.check_viscosity(660, 2.54e7, 3.8212e-5, 1e-4)
-        self.check_viscosity(660, 2.8e7, 5.3159e-5, 1e-4)
-
-    def test_thermal_conductivity_supercritical(self):
-        self.check_thermal_conductivity(660, 2.2e7, 0.14872, 1e-4)
-        self.check_thermal_conductivity(660, 2.54e7, 0.35484, 1e-4)
-        self.check_thermal_conductivity(660, 2.8e7, 0.38479, 1e-4)
-
-
-class TestTransportData(utilities.CanteraTest):
-    @classmethod
-    def setUpClass(cls):
-        utilities.CanteraTest.setUpClass()
-        cls.gas = ct.Solution("h2o2.yaml")
-        cls.gas.X = 'H2O:0.6, H2:0.4'
-
-    def test_read(self):
-        tr = self.gas.species('H2').transport
-        self.assertEqual(tr.geometry, 'linear')
-        self.assertNear(tr.diameter, 2.92e-10)
-        self.assertNear(tr.well_depth, 38.0 * ct.boltzmann)
-        self.assertNear(tr.polarizability, 0.79e-30)
-        self.assertNear(tr.rotational_relaxation, 280)
-
-    def test_set_customary_units(self):
+
+    @pytest.fixture(scope='class')
+    def water(self):
+        return ct.Solution('thermo-models.yaml', 'liquid-water')
+
+    @pytest.mark.parametrize("T, P, mu, rtol", [
+        (400, 1e6, 2.1880e-4, 2e-4),
+        (400, 8e6, 2.2061e-4, 2e-4),
+        (620, 1.6e7, 6.7489e-5, 1e-4),
+        (620, 2.8e7, 7.5684e-5, 1e-4),
+    ])
+    def test_viscosity_liquid(self, water, T, P, mu, rtol):
+        water.TP = T, P
+        assert water.viscosity == approx(mu, rel=rtol)
+
+    @pytest.mark.parametrize("T, P, mu, rtol", [
+        (660, 2.2e7, 2.7129e-5, 1e-4),
+        (660, 2.54e7, 3.8212e-5, 1e-4),
+        (660, 2.8e7, 5.3159e-5, 1e-4),
+    ])
+    def test_viscosity_supercritical(self, water, T, P, mu, rtol):
+        water.TP = T, P
+        assert water.viscosity == approx(mu, rel=rtol)
+
+    @pytest.mark.parametrize("T, P, k, rtol", [
+        (400, 1e6, 0.68410, 1e-4),
+        (400, 8e6, 0.68836, 1e-4),
+        (620, 1.6e7, 0.45458, 1e-4),
+        (620, 2.8e7, 0.49705, 1e-4),
+    ])
+    def test_thermal_conductivity_liquid(self, water, T, P, k, rtol):
+        water.TP = T, P
+        assert water.thermal_conductivity == approx(k, rel=rtol)
+
+    @pytest.mark.parametrize("T, P, k, rtol", [
+        (660, 2.2e7, 0.14872, 1e-4),
+        (660, 2.54e7, 0.35484, 1e-4),
+        (660, 2.8e7, 0.38479, 1e-4),
+    ])
+    def test_thermal_conductivity_supercritical(self, water, T, P, k, rtol):
+        water.TP = T, P
+        assert water.thermal_conductivity == approx(k, rel=rtol)
+
+
+class TestTransportData:
+
+    @pytest.fixture(scope='class')
+    def gas(self):
+        gas = ct.Solution("h2o2.yaml")
+        return gas
+
+    @pytest.fixture(scope='function', autouse=True)
+    def initial_condition(self, gas):
+        gas.X = 'H2O:0.6, H2:0.4'
```

Comment:
While the old setup method did this, I don't think it has any effect for the tests done here.